src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/ConvictionRestClient.java

```diff
@@ -43,7 +44,7 @@
     private String convictionUrlTemplate;
 
     @Value("${community-api.current-order-header-url-template}")
```

Comment:
Should we rename the variable here ? current-order-header-url-template does not match the `sentenceStatusUrlTemplate`

Reply:
Good spot, will do